runtime/doc/various.txt

```diff
@@ -211,10 +211,8 @@ g8			Print the hex values of the bytes used in the
 :te[rminal][!]		Open an interactive shell in a |terminal-emulator|.
 			Use the shell specified in 'shell'.
 
-			The new terminal buffer takes over the current split
-			window (like |:edit|). This fails when changes have
-			been made to the current buffer, unless 'hidden' is
-			set.
+			The new terminal buffer takes over the current window
```

Comment:
Hmm, now you did the opposite of what I had in mind. :-P

I mean, a new buffer acts in the current window by default, so I thought it shouldn't be mentioned. But the remark about unsaved changes is quite important.

Moreover, as far as I recall, "taking over a window" is a wording that isn't used anywhere else in the docs, that's why I meant it should be left away or be merged with the following sentence:

Something like this:

```
Creating the terminal buffer fails when changes have been
made to the current buffer, unless 'hidden' is set.
```

FWIW, usually Justin chimes in for the final wording, so maybe wait a moment before you change it yet again. :-)